src/openrct2-ui/windows/EditorObjectSelection.cpp

```diff
@@ -105,10 +105,10 @@ static constexpr const ObjectPageDesc ObjectSelectionPages[] = {
     { STR_OBJECT_SELECTION_WATER,                       SPR_TAB_WATER,              false },
 
     // Currently hidden until new save format arrives:
-    // { STR_OBJECT_SELECTION_TERRAIN_SURFACES,            SPR_G2_TAB_LAND,            false },
-    // { STR_OBJECT_SELECTION_TERRAIN_EDGES,               SPR_G2_TAB_LAND,            false },
-    // { STR_OBJECT_SELECTION_STATIONS,                    SPR_TAB_PARK,               false },
-    // { STR_OBJECT_SELECTION_MUSIC,                       SPR_TAB_MUSIC_0,            false },
+    { STR_OBJECT_SELECTION_TERRAIN_SURFACES,            SPR_G2_TAB_LAND,            false },
```

Comment:
Enabling these tabs would be fine in my book, as long as we don't allow changes to these tabs. It means that people are aware of the possibilities.

Reply:
I didn't mean for this to be included in the PR, I just wanted to test it.